sw/device/rom_exts/manifest.md

```diff
@@ -358,3 +366,77 @@ B, without having to re-link or re-compile the image (note we only have one
 entry address). At least initially, we will only create images that can work
 from Slot A, and later we will add support ensuring these images are linked in
 a way that they can run from Slot B without modification.
+
+
+## How To Validate a ROM_EXT Image
+
+Below we provide an abstract definition of how to sign and verify a ROM_EXT
+image. This description is provided so that there is a specification of what the
+signing and validation systems are implementing, so that alternate
+implementations may be produced if necessary.
+
+Notation:
+
+*   `rom_ext` - ROM_EXT Image, including manifest and contents (Variable Length)
+    `signed_area(rom_ext)` denotes just the area between the start of the signed
+    contents and the end of the image.
+*   `device_usage_value` - Device-Calculated Usage Constraint Value (1024
+    bytes). This is the "device usage value" described above, as calculated by
+    the Mask ROM.
+*   `system_state_value` - Device System State Value (32 bytes). This is a Mask
+    ROM calculated value that summarises the current device state.
+*   `private_key` - RSA Private Key
+*   `public_key` - RSA Public Key
+*   `signature` - ROM_EXT RSA Signature,
+*   `||` - Denotes concatenation without a delimiter.
+*   `:=` - Denotes assignment.
+
+### Algorithms
+
+*   We primarily use
+    [`RSASSA-PKCS1-V1_5-SIGN`](https://tools.ietf.org/html/rfc3447#section-8.2.1)
+    and
+    [`RSASSA-PKCS1-V1_5-VERIFY`](https://tools.ietf.org/html/rfc3447#section-8.2.2)
+    for signatures and verification.
+*   RSA Signing and Verification is done with 3072-bit keys.
+*   The `Hash` Operation may be done with SHA2-265, SHA2-384, or SHA2-512,
```

Comment:
There is only hardware support for `SHA2-256`, `SHA3-256`, `SHA3-384` and `SHA3-512`. 

Can you add a note stating that the HASH is required to provide a security level equivalent to the one provided by the signature?

Reply:
Done